### ISSUE TITLE
Yuzu Android not recognizing BSP-D3 gamepad

### ISSUE THREAD

**Issue by kjanek** [yuzu-emu/yuzu#11024]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Early access 1462db4694 Android

### Description of Issue

The emulator doesn't recognize the gamepad after entering game (can use it while in grid games list). It looks like if it would get the first interaction but maybe not the specific buttons cause i can go to the loding menu of mario but thats all. Its a Gamepad with multiple recognition modes. Have tested it at Xinput mode and normal Android Gamepad mode. It doesnt work in game.
![obraz](https://github.com/yuzu-emu/yuzu/assets/25937163/43ab9e9a-f05e-4d7a-9ee6-d2489a53775d)
![obraz](https://github.com/yuzu-emu/yuzu/assets/25937163/c8029393-bfe8-4566-80cf-530f4124ddf9)
![obraz](https://github.com/yuzu-emu/yuzu/assets/25937163/1a175428-25a7-4c54-bc86-440b47c3dc62)
![obraz](https://github.com/yuzu-emu/yuzu/assets/25937163/b5e82f0a-caee-4f0d-9430-fbbefd345ab3)
![obraz](https://github.com/yuzu-emu/yuzu/assets/25937163/f4284c18-ae55-4fda-8afe-1045925d2b84)

### Expected Behavior

Be recognized in game or be able to bind the keys

### Reproduction Steps

connect pad

### Log File

[yuzu_log (5).txt](https://github.com/yuzu-emu/yuzu/files/11952498/yuzu_log.5.txt)

### System Configuration

CPU: Snapdragon 865
GPU/Driver: Adreno 650
RAM: 6gb
OS: Android 12 - Lineage os 19.1

**Comment by german77** [comment]
Does it work for other android apps? We use native support of android for controller support

**Comment by kjanek** [comment]
Yes it works perfectly fine in other apps - tested mainly in PPSSPP and minecraft

**Comment by kjanek** [comment]
I checked with another Switch game (Celeste) and its working no problem. I am 100% sure it wasnt working the previous update. I guess it must be something with Mario. If I can I will check some more games and report back.

**Comment by german77** [comment]
Oh so it's an emulation issue and not the controller itself. Good to know.

**Comment by kjanek** [comment]
I checked Army of ruin and it works too.
So for now I am seeing this behavior only in mario.

**Comment by inshua** [comment]
> I checked Army of ruin and it works too. So for now I am seeing this behavior only in mario.

yes, I met it too
